tests/book-reader.js

```diff
@@ -0,0 +1,44 @@
+// automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
I don't think these files should be directly in `tests`, they aren't for any other languages.